### PR TITLE
Update the error burst alert runbooks

### PR DESCRIPTION
**Pull request by sradco**
Updated the alert runbooks related to error burst alerts, to indicate the burst in ongoing for the last 5 minutes and perhaps event more.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

Fixes: https://issues.redhat.com/browse/CNV-38540

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by sradco** [comment]
@machadovilaca @assafad please review

**Comment by assafad** [comment]
I could see we use 5m twice in those alerts definitions, and I'm trying to understand what it means, please correct me if I'm wrong:
- In the alert expression - IIUC, the condition is met if within the past 5m at least 80% of the requests have failed.
- in the `For:` - IIUC, this means the alert starts firing if the expression is constantly met for 5m (and before that the alert is pending).

If the above is true, it means that once such an alert is firing, there is 80% errors for at least 10 minutes.

**Comment by sradco** [comment]
> I could see we use 5m twice in those alerts definitions, and I'm trying to understand what it means, please correct me if I'm wrong:
> 
> * In the alert expression - this means the condition is met if within the past 5m at least 80% of the requests have failed.
> * in the `For:` - this means the alert starts firing if the expression is constantly met for 5m (and before that the alert is pending).
> 
> If the above is true, it means that such alert is firing when there are 80% errors within at least 10 minutes.

@assafad correct. Thank you. Updated.

File: docs/runbooks/VirtApiRESTErrorsBurst.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # VirtApiRESTErrorsBurst
-<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
 
 ## Meaning
 
-More than 80% of REST calls have failed in the `virt-api` pods in the last 5 minutes.
+For at least the last 10 minutes, more than 80% of the REST calls to `virt-api` pods have been failing consistently.
```
Comment (assafad) [inline docs/runbooks/VirtApiRESTErrorsBurst.md:5]: nit: `For at least the last 10 minutes` -> `For the past/last 10 minutes or longer, ...`

Comment (sradco) [reply docs/runbooks/VirtApiRESTErrorsBurst.md]: @assafad  Updated

File: docs/runbooks/VirtApiRESTErrorsBurst.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
 # VirtApiRESTErrorsBurst
-<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
 
 ## Meaning
 
-More than 80% of REST calls have failed in the `virt-api` pods in the last 5 minutes.
+For the last 10 minutes or longer, over 80% of the REST calls made to `virt-api` pods have consistently failed.
```
Comment (ousleyp) [inline docs/runbooks/VirtApiRESTErrorsBurst.md:5]: - Is "consistently" referring to "over 80%"? I think "over 80%" is sufficient unless "consistently" is referring to something else.
- Does the alert fire every 10 minutes? If so, I don't think we need to specify "for the last 10 minutes or longer." While that might be literally true, it makes more sense to me to limit the message to the actual scope of the alert.

Would the following phrasing be sufficient? 

```suggestion
During the past 10 minutes, more than 80% of REST calls made to `virt-api` pods have failed.
```

**Review by ousleyp — COMMENTED** [review]
Hi @sradco, just one comment, but it is applicable to all of the files. Otherwise, LGTM. Thanks!

File: docs/runbooks/VirtApiRESTErrorsBurst.md (hunk 2 above)
Comment (sradco) [reply docs/runbooks/VirtApiRESTErrorsBurst.md:5]: @ousleyp We calculate the % of failed  REST calls in the last 5 minutes and if we see that it stays over 80% for additional 5 minutes, the alert will start firing. 
So in Total we can say we have the issue for 10 minutes or more.

The alert will continue firing until the issue is fixed and the % of the  REST calls in the last 5 minutes will be below 80%.

It means there is a very serious issue and most of the  REST calls fail.
WDYT?

**Pull request merged**